### PR TITLE
Document dynamic text option

### PR DESCRIPTION
**Pull request by davekempe**
We forgot about a the dynamic text card type

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -81,6 +81,9 @@ Select a set of images you want to toggle between on state change.
 #### Icinga Line
 Allows you to draw lines, you can rotate and resize them, and set the weight. They only toggle between OK, Warning, Critical, Unknown at present.
 
+#### Dynamic Text
+Allows you to pull text out of a host_var or service_var and display it on the dashboard like a card.
```
Comment (ollytom) [inline README.md:85]: host_var or service_var - can we just the regular words "Host or Service variables"

**Review by ollytom — CHANGES_REQUESTED** [review]
The commit message should contain what we're actually doing e.g. "Document dynamic text element".

File: README.md (hunk 1 above)
Comment (davekempe) [reply README.md:85]: thanks for the feedback.

Comment (davekempe) [reply README.md]: @ollytom updated for your review.

**Pull request closed without merging**